ample/main.py

```diff
@@ -509,6 +470,29 @@ def setup(self, optd):
         logger.info('All needed programs are found, continuing...')
         return optd
 
+    def setup_workdir(self, argso):
+        # Make a work directory - this way all output goes into this directory
+        if argso['work_dir'] and not argso['restart_pkl']:
+            print('Making a named work directory: %s', argso['work_dir'])
+            try:
+                os.mkdir(argso['work_dir'])
+            except Exception as e:
+                msg = "Cannot create work_dir {0}: {1}".format(argso['work_dir'], e)
+                exit_util.exit_error(msg, sys.exc_info()[2])
+
+        if not argso['work_dir']:
+            if not os.path.exists(argso['run_dir']):
+                msg = 'Cannot find run directory: {0}'.format(argso['run_dir'])
+                exit_util.exit_error(msg, sys.exc_info()[2])
+            if bool(argso['rvapi_document']):
```

Comment:
why the bool casting? I'd say if
```
if argso['rvapi_document'] is not None:
```

Reply:
I think this is more flexible and potentially more explicit?

Reply:
no, and it's also not pythonic. if you expect a bool, then no need to typecast. otherwise python's implementation is redundant.

probs should be `if args['rvapi_document']:`

Reply:
agreed